app/components/primer/box_component.rb

```diff
@@ -3,6 +3,12 @@
 module Primer
   # A basic wrapper component for most layout related needs.
   class BoxComponent < Primer::Component
+    # @example 20|Default
+    #   <%= render(Primer::BoxComponent.new) { "Your content here" } %>
+    #
+    # @example 20|Color and padding
```

Comment:
The height of the example will need to be larger because of the extra padding :)

```suggestion
    # @example 54|Color and padding
```